samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/DocumentImportController.cs

```diff
@@ -115,6 +135,13 @@ private enum SupportedFileType
                 case SupportedFileType.Pdf:
                     fileContent = this.ReadPdfFile(formFile);
                     break;
+                case SupportedFileType.Jpg:
+                case SupportedFileType.Png:
```

Comment:
Please remove code for Jpg and Png if we are not planning to support them yet.

Reply:
Both these formats are supported now with the changes made.  Jpg, Png and Tiff all will call the "ReadTextFromImageFileAsync" method to set the "fileContent" variable.